### PR TITLE
Upgrade rootstock on 2019-06-03

### PR DESCRIPTION
**Pull request by dhimmel**
Greetings, I'm Daniel, a developer of Manubot. Thanks for trying it out as we iron out all the issues and improve user experience. We periodically create PRs like this to help users upgrade to latest versions, if that is something they would like.

This pull request upgrades your Manubot version using [these instructions](https://github.com/manubot/rootstock/blob/dcb3cc43880694acf4ea25774fabe328f20c3adb/SETUP.md#merging-upstream-rootstock-changes). Note the following about merging:

> If updating master via a pull request, proceed to push the commit to GitHub and open a pull request. Once the pull request is ready to merge, use GitHub's "Create a merge commit" option rather than "Squash and merge" or "Rebase and merge" to preserve the rootstock commit hashes.

In the next comment, I'll upload the PDF generated from the latest manubot.

### CODE REVIEW

**Comment by dhimmel** [comment]
Here's the PDF output: 
[manuscript.pdf](https://github.com/jmonlong/manu-vgsv/files/3249123/manuscript.pdf).

The web view will have improved interactivity like can be seen at https://greenelab.github.io/meta-review/

File: ci/deploy.sh [hunk 1]
```diff
@@ -49,8 +49,8 @@ This build is based on
 https://github.com/$TRAVIS_REPO_SLUG/commit/$TRAVIS_COMMIT.
 
 This commit was created by the following Travis CI build and job:
-https://travis-ci.org/$TRAVIS_REPO_SLUG/builds/$TRAVIS_BUILD_ID
-https://travis-ci.org/$TRAVIS_REPO_SLUG/jobs/$TRAVIS_JOB_ID
+https://travis-ci.com/$TRAVIS_REPO_SLUG/builds/$TRAVIS_BUILD_ID
+https://travis-ci.com/$TRAVIS_REPO_SLUG/jobs/$TRAVIS_JOB_ID
```
Comment (dhimmel) [inline ci/deploy.sh]: Looks like this repo uses travis-ci.com, so this will fix the auto-generated commit messages.

**Comment by jmonlong** [comment]
Thanks for preparing this @dhimmel!

**Comment by dhimmel** [comment]
Rendered manuscript in https://jmonlong.github.io/manu-vgsv/v/be17ff667686a0717fec21a8b1b76bae032b729d/. Let us know if there are any issues.

**Pull request merged**